_maps/pubbystation.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
`There are 10 active turfs at roundstart caused by a difference of the air between the adjacent turfs. You can see its coordinates using "Mapping -> Show roundstart AT list" verb (debug verbs required).`
<hr>
This is caused by these tiles on the AI satellite (inside the red boxes) having an atmosphere on them when the round starts, so they're immediately being depressurised and causing unnecessary atmos processing.

![image](https://user-images.githubusercontent.com/57483089/169620863-f93d74f3-7bda-4754-93f8-fdba5e698b33.png)
The fix is to change all of the floor tiles on those sections from `/turf/open/floor/plating` to `/turf/open/floor/plating/airless`.